modules/openapi-generator/src/main/resources/dart2/apilib.mustache

```diff
@@ -24,14 +24,18 @@ part 'auth/http_bearer_auth.dart';
 {{#models}}{{#model}}part 'model/{{{classFilename}}}.dart';
 {{/model}}{{/models}}
 
+/// A handy comparison utility to deep-compare any arbitrary objects.
+const deepEquality = DeepCollectionEquality();
```

Comment:
is this necessary ?

Reply:
Probably not, given that Dart's `const`s are compiled once. I'll revert.

Reply:
✅